resource/resource_test.go

```diff
@@ -695,6 +695,21 @@ Cogito SOURCE configuration:
 			repoURL:     "foo://bar",
```

Comment:
Typo in the commit message title? `repro`?

Reply:
No, "repro" is a shorthand for "reproduction", a setup that can reproduce a bug.

Reply:
I thought so, but I was not 100% sure.